db/migrate/20230919130604_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :AuthorId
+      t.integer :PostId
```

Comment:
- [ ] @Kgomotso196 Great work in your comments table, however, I would recommend the removal of these two columns as they will be automatically generated when you run your migration(using add_Refernece which automatically adds the column and index to the foreign key )

Reply:
Thank you for your constructive feedback @danmainah.